crates/holochain/tests/dht_arc.rs

```diff
@@ -40,15 +40,15 @@ async fn test_arc_coverage() {
                 let p = peers.clone();
                 let arc = peers.get_mut(i).unwrap();
                 let bucket = DhtArcBucket::new(*arc, p.clone());
-                let density = bucket.density();
+                let density = bucket.peer_view_alpha();
                 arc.update_length(density);
             }
             let bucket = DhtArcBucket::new(
                 DhtArc::new(MAX_HALF_LENGTH / 2, MAX_HALF_LENGTH),
                 peers.clone(),
             );
             println!("{}", bucket);
-            std::thread::sleep(std::time::Duration::from_millis(200));
+            // std::thread::sleep(std::time::Duration::from_millis(200));
```

Comment:
```suggestion
```

Reply:
This entire test is removed in a subsequent PR